django/apps/config.py

```diff
@@ -139,7 +139,11 @@ def create(cls, entry):
                 "'%s' must supply a name attribute." % entry)
 
         # Ensure app_name points to a valid module.
-        app_module = import_module(app_name)
+        try:
+            app_module = import_module(app_name)
+        except ImportError:
+            raise ImproperlyConfigured(
+                "Cannot import '%s', check AppConfig.name is correctly defined.")
```

Comment:
There is a `%s` in the exception message, but you forgot to pass `app_name`.

A minor code style suggestion:

```py
raise ImproperlyConfigured(
    "Cannot import '%s', check AppConfig.name is correctly defined." % app_name
)
```